_Sidebar.md

```diff
@@ -15,6 +15,7 @@
 **Developing Oppia**
   * [[FAQs|Frequently-Asked-Questions]]
   * [[Installing Oppia|https://github.com/oppia/oppia/wiki/Installing-Oppia]]
+  * [[Tips for common editors|Tips-for-common-editors]]
```

Comment:
Maybe "editors" --> "IDEs"? When I read "editors" I think of people.

Reply:
done